File: src/main/java/com/github/davinkevin/transmissionrss/transmission/config/JacksonConfig.java
package com.github.davinkevin.transmissionrss.transmission.config;

import com.fasterxml.jackson.databind.ObjectMapper;
import com.fasterxml.jackson.databind.SerializationFeature;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

/**
 * Created by kevin on 08/04/2018
 */
@Configuration
public class JacksonConfig {

    @Bean
    ObjectMapper mapper() {
        return new ObjectMapper()
                .disable(SerializationFeature.FAIL_ON_EMPTY_BEANS);
    }

}

File: src/main/kotlin/com/github/davinkevin/transmissionrss/transmission/config/JacksonConfig.kt
package com.github.davinkevin.transmissionrss.transmission.config

import com.fasterxml.jackson.databind.ObjectMapper
import com.fasterxml.jackson.databind.SerializationFeature
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class JacksonConfig {

    @Bean
    fun mapper() = ObjectMapper()
            .disable(SerializationFeature.FAIL_ON_EMPTY_BEANS)
}